src/sections/Projects/Sistent/sistent.style.js

```diff
@@ -21,6 +21,9 @@ const SistentWrapper = styled.div`
   .content {
     width: 100%;
     padding-bottom: 2rem;
+    @media only screen and (max-width: 425px){
```

Comment:
why we are not opening the dropdown above max width 425, it should open until the dropdown is displaying

Reply:
> why we are not opening the dropdown above max width 425, it should open until the dropdown is displaying

It will appear, this query is for **.content**  not for dropdown. It is basically making **.content overflow** to be **hidden** on smaller screens so that dropdown can work. You can verify that via commit preview url.